<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="[HBS] Edit Delivered Mail(Process + Validation)"
           tests="5"
           failures="2"
           errors="0"
           skipped="0"
           time="110.210">
    <properties></properties>

            <testcase
                name="TC_16_upload_maximum_capacity_multi_file"
                time="18.259"
                classname="test.Mail.Edit_Delivered_Mail">

            <failure type="java.lang.AssertionError" message="The following asserts failed:
	[Failed] Can not upload multi file with capacity = 2MB.
 Actual message: 2MBを超えるメールを配信することはできません。
 Expect message: contains ファイルがアップロードされました expected [true] but found [false]">
                <![CDATA[java.lang.AssertionError: The following asserts failed:
	[Failed] Can not upload multi file with capacity = 2MB.
 Actual message: 2MBを超えるメールを配信することはできません。
 Expect message: contains ファイルがアップロードされました expected [true] but found [false]
                org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)page.Mail.Step2_Attachment.upload_maximum_capacity_multi_file(Step2_Attachment.java:194)test.Mail.Edit_Delivered_Mail.TC_16_upload_maximum_capacity_multi_file(Edit_Delivered_Mail.java:251)java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)java.base/java.lang.reflect.Method.invoke(Method.java:568)org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)java.base/java.util.ArrayList.forEach(ArrayList.java:1511)org.testng.TestRunner.privateRun(TestRunner.java:808)org.testng.TestRunner.run(TestRunner.java:603)org.testng.SuiteRunner.runTest(SuiteRunner.java:429)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)org.testng.SuiteRunner.run(SuiteRunner.java:326)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)org.testng.TestNG.runSuitesLocally(TestNG.java:1169)org.testng.TestNG.runSuites(TestNG.java:1092)org.testng.TestNG.run(TestNG.java:1060)com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </failure>
    </testcase>
            <testcase
                name="TC_15_upload_maximum_capacity_1_file"
                time="7.101"
                classname="test.Mail.Edit_Delivered_Mail">

            <failure type="java.lang.AssertionError" message="The following asserts failed:
	[Failed] Can not upload file with capacity: 2MB.
 Actual message: 2MBを超えるメールを配信することはできません。
 Expect message: contains ファイルがアップロードされました expected [true] but found [false]">
                <![CDATA[java.lang.AssertionError: The following asserts failed:
	[Failed] Can not upload file with capacity: 2MB.
 Actual message: 2MBを超えるメールを配信することはできません。
 Expect message: contains ファイルがアップロードされました expected [true] but found [false]
                org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)page.Mail.Step2_Attachment.upload_maximum_capacity_1_file(Step2_Attachment.java:161)test.Mail.Edit_Delivered_Mail.TC_15_upload_maximum_capacity_1_file(Edit_Delivered_Mail.java:239)java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)java.base/java.lang.reflect.Method.invoke(Method.java:568)org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)java.base/java.util.ArrayList.forEach(ArrayList.java:1511)org.testng.TestRunner.privateRun(TestRunner.java:808)org.testng.TestRunner.run(TestRunner.java:603)org.testng.SuiteRunner.runTest(SuiteRunner.java:429)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)org.testng.SuiteRunner.run(SuiteRunner.java:326)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)org.testng.TestNG.runSuitesLocally(TestNG.java:1169)org.testng.TestNG.runSuites(TestNG.java:1092)org.testng.TestNG.run(TestNG.java:1060)com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </failure>
    </testcase>


            <testcase
                name="TC_18_upload_exceed_maximum_capacity_multi_file"
                time="17.649"
                classname="test.Mail.Edit_Delivered_Mail">
            </testcase>
            <testcase
                name="TC_19_upload_exceed_10_files"
                time="52.409"
                classname="test.Mail.Edit_Delivered_Mail">
            </testcase>
            <testcase
                name="TC_17_upload_exceed_maximum_capacity_1_file"
                time="14.792"
                classname="test.Mail.Edit_Delivered_Mail">
            </testcase>
</testsuite>
